pmm-app/src/pmm-check/CheckPanel.tsx

```diff
@@ -76,7 +79,14 @@ export const CheckPanel: FC = () => {
             {tabs.map((tab) => tab.key === activeTab && tab.component)}
           </TabContent>
         </>
-      )}
+      ) : (
+        <div className={styles.empty} data-qa="db-check-panel-settings-link">
+          Security Threat Tool is disabled. You can enable it in&nbsp;
+          <Link className={styles.link} to={PMM_SETTINGS_URL}>
+            PMM Settings.
```

Comment:
And this too. As two separate strings.

Reply:
Done [in this PR](https://github.com/percona/grafana-dashboards/pull/1109)